kafka-client/src/main/java/dev/responsive/kafka/internal/stores/ResponsiveWindowStore.java

```diff
@@ -110,7 +110,9 @@ public void init(final StateStoreContext storeContext, final StateStore root) {
           window -> window.windowStartMs >= minValidTimestamp()
       );
 
-      numBloomFilterWindows = config.getInt(WINDOW_BLOOM_FILTER_COUNT_CONFIG);
+      numBloomFilterWindows = params.retainDuplicates()
+          ? 0 // disable bloom filters for stream-stream join/duplicate stores
```

Comment:
why don't bloom filters work in this scenario? we should be able to know if there are no values in the remote, right?

Reply:
there's a few reasons but the main one is that stream-stream joins only use range queries whereas the bloom filters are only implemented for/help with the point lookups.

I suppose you could argue we should still enable them in case someone is using a window store with duplicates from a custom processor, which is rare but theoretically valid. However the point lookup API is essentially not defined for duplicate window stores due to the restrictions of the store interface, so I really doubt anyone is ever using point lookups in this case (and if they are, I'm highly skeptical of what they're doing and whether it should just be a regular/non-duplicate store)